client/src/utils/tone/editor-notes.ts

```diff
@@ -0,0 +1,47 @@
+export const editorNotes = [
```

Comment:
Could we generate this from editor-config?

Reply:
We could put the array in that file instead of its own, but this array determines the 'tune' that plays so it can't be generated.

Reply:
Oh, my mistake. Thanks for explaining.